editor/editor_inspector.cpp

```diff
@@ -1563,14 +1566,22 @@ void EditorInspectorCategory::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_THEME_CHANGED: {
-			if (menu) {
-				if (is_favorite) {
-					menu->set_item_icon(menu->get_item_index(EditorInspector::MENU_UNFAVORITE_ALL), get_editor_theme_icon(SNAME("Unfavorite")));
-				} else {
-					menu->set_item_icon(menu->get_item_index(MENU_OPEN_DOCS), get_editor_theme_icon(SNAME("Help")));
-				}
+			menu_icon_dirty = true;
+			_update_icon();
+		} break;
+
+		case NOTIFICATION_TRANSLATION_CHANGED: {
+			if (is_favorite) {
+				label = TTR("Favorites");
+			}
+
+			RID ae = get_accessibility_element();
+			if (ae.is_valid()) {
+				DisplayServer::get_singleton()->accessibility_update_set_name(ae, vformat(TTR("Category: %s"), label));
+				DisplayServer::get_singleton()->accessibility_update_set_value(ae, vformat(TTR("Category: %s"), label));
```

Comment:
Accessibility values should not be updated from any place except `NOTIFICATION_ACCESSIBILITY_UPDATE` (works the same way as `NOTIFICATION_DRAW` and `draw_` calls work).

When it needs updating, call `queue_accessibility_update();` instead.